src/message/messageActionSheet.js

```diff
@@ -38,8 +38,8 @@ export type ShowActionSheetWithOptions = (
 
 type TopicArgs = {
   auth: Auth,
-  stream: string,
   topic: string,
+  streamId?: number,
```

Comment:
This shouldn't be optional, we should throw an error before we construct this if there's no stream ID.

Reply:
I have added a try catch block before `showTopicActionSheet` in the new revision and made `stream_id` required.
In case try fails I am also displaying a toast with a generic "Something went wrong". I can't figure out a message concise enough to display here so that the user gets an idea what went wrong so I went with that for now, if you have any suggestions let me know.